portfoliostyles.css

```diff
@@ -0,0 +1,126 @@
+*{
+	box-sizing: border-box;
+}
+
+html{
+	font-size: 62.5%;
+}
+
+body{
+ background-color: #16262E;
+ background-image: url("code.jpeg");
+ font-family: "Courier New";
+}
+
+a:link{
+	text-decoration: none;
+	color: #ffffff;
+}
+
+a:visted{
+	text-decoration: none;
+	color: #ffffff;
+}
+
+.NavBar{
```

Comment:
change all tags / properties in both the HTML and CSS to 'camel case'
e.g. navBar